testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-tls-kerberos/config-master.properties

```diff
@@ -3,7 +3,7 @@ node.environment=test
 node.internal-address-source=FQDN
 
 coordinator=true
-node-scheduler.include-coordinator=true
```

Comment:
That was intentional.

Scheduling _also_ on coordinator makes the test setup "harder to pass" 

Without this, some communication lines won't be fully exercised.

What was the rationale for this change? The commit message included none.

I found this in the PR description

> If the query could be scheduled on coordinator - it could be enough for successful query execution. The whole environment could pretend that it's multinode but in reality could be singlenode with wrongly configured worker.

but the scheduler doesn't favor the coordinator-as-a-worker, so i don't fully get it

Reply:
With this if worker is unable to join we would see that query is still passing. So we would get false positive as full setup failed.

Reply:
good point; this can be alleviated by using min worker count setting